spec/mspec/lib/mspec/utils/script.rb

```diff
@@ -73,7 +73,12 @@ def try_load(target)
 
     names.each do |name|
       config[:path].each do |dir|
-        file = File.expand_path name, dir
+        begin
+          file = File.expand_path name, dir
+        rescue ArgumentError
+          # File.expand_path can issue error e.g. if HOME is not available
+          return false
```

Comment:
It can raise an `ArgumentError` when `dir` has `~/` too, in theory.

```suggestion
          next
```